core/tests/test_utils.py

```diff
@@ -168,6 +196,13 @@ def log_line(self, line):
         print '%s%s' % (LOG_LINE_PREFIX, line)
 
     def _delete_all_models(self):
+        """Deactivates environment of Test Base
```

Comment:
This function is doing exactly the same thing as tearDown(self) function above ?